Core.ApplicationServices/Users/Write/UserWriteService.cs

```diff
@@ -14,6 +14,7 @@
 using Core.DomainModel.Organization;
 using Core.DomainServices;
 using Core.DomainServices.Generic;
+using dk.nita.saml20.Schema.Protocol;
```

Comment:
Remove the unused reference (this is the package we're using to authenticate SSO calls with Kombit - you're giving me PTSD)